Clicker/Supporting/Constants.swift

```diff
@@ -88,7 +88,7 @@ struct RequestKeys {
     static let answerIdKey = "answerId"
     static let choiceKey = "choice"
     static let countKey = "count"
-    static let googleIdKey = "googleId"
+    static let googleIdKey = "googleID"
```

Comment:
can you replace `googleIdKey` to be `googleIDKey` just to reflect what backend does